src/detection/PsfFormatter.cc

```diff
@@ -22,14 +22,13 @@
 #include "lsst/daf/persistence/BoostStorage.h"
 #include "lsst/daf/persistence/XmlStorage.h"
 #include "lsst/pex/exceptions.h"
-#include "lsst/pex/logging/Trace.h"
+#include "lsst/log/Log.h"
 #include "lsst/pex/policy/Policy.h"
 
 BOOST_CLASS_EXPORT(lsst::afw::detection::Psf)
 
-#define EXEC_TRACE  20
-static void execTrace(std::string s, int level = EXEC_TRACE) {
-    lsst::pex::logging::Trace("afw.detection.PsfFormatter", level, s);
+namespace {
+LOG_LOGGER _log = LOG_GET("afw.detection.PsfFormatter");
```

Comment:
Ah, got it - I wasn't considering the guarantees about ordering that come from the linker.  I think the current approach is fine, then.  Thanks!